packages/dd-trace/test/appsec/index.next.plugin.spec.js

```diff
@@ -146,7 +152,7 @@ describe('test suite', () => {
       }
     ]
 
-    if (satisfies(realVersion, '>=13.2')) {
+    if (satisfies(realVersion, '>=13.2') && !(NODE_MAJOR >= 24 && satisfies(realVersion, '=13.2'))) {
```

Comment:
I personally find negations always more tricky to read than the other way around. So I would have written it as:

```suggestion
    if (satisfies(realVersion, '>=13.2') && (NODE_MAJOR < 24 || satisfies(realVersion, '!=13.2'))) {
```

Reply:
I have no strong opinion about one or other... suggestion addressed.